### PR TITLE
Jquery fix

### PR DESCRIPTION
**Pull request by katlings**

### CODE REVIEW

**Comment by katlings** [comment]
@cephalopodblue

File: djangocms_youtube/forms.py [hunk 1]
```diff
@@ -17,6 +17,9 @@ class Meta:
             'video_url': YoutubeVideoURLWidget(),
         }
 
+    class Media:
+        js = ('admin/js/jquery.init.js',)
```
Comment (katlings) [inline djangocms_youtube/forms.py:21]: defining the jquery like this is a "must" according to the documentation, but just adding this class didn't seem to fix the problem where django.jQuery wasn't being recognized. https://docs.djangoproject.com/en/3.2/ref/contrib/admin/#contrib-admin-jquery

File: djangocms_youtube/static/admin/js/djangocms_youtube/changeform.js [hunk 2]
```diff
@@ -1,69 +1,71 @@
-(function($) {
-    $(function() {
+window.addEventListener("load", function () {
```
Comment (katlings) [inline djangocms_youtube/static/admin/js/djangocms_youtube/changeform.js:1]: This file is untouched aside from this wrapper. I have no idea how it was working before.

**Review by cephalopodblue — APPROVED** [review]
This is really odd. jQuery has `$( document ).ready()`, which I think is doing what you have here but... with jQuery... which isn't available at this point (or at least that's what the error we get here seems to be saying :/)

I don't have any better ideas re: what's going on, and this is a fix that works, so: LGTM.

**Comment by katlings** [comment]
jQuery is actually available at this point, but only because we're loading it at some other point in the broader webpage. It's django's packaged and namespaced jQuery that's not available. We have other pages on kexp.org successfully using django.jQuery without this trick, so I've been putting off the merge in case I can figure out what was happening, but nothing's come to mind.... this seems like a dumb but harmless fix.

**Pull request merged**